content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -123,6 +123,34 @@ const prisma = new PrismaClient({
 })
 ```
 
+### <inlinecode>adapter</inlinecode>
+
+Defines an instance of a driver adapter. See also [Drivers](/concepts/components/drivers#driver-adapters) <span class="concept"></span>.
+
+This is available from version 5.4.0 and newer. 
+
+This feature is currently in Preview behind the `driverAdapters` feature flag.
+
+#### Example
+
+The example below uses the [Neon driver adapter](/concepts/components/drivers#neon):
```

Comment:
Invalid link, should go to Neon page instead.

Reply:
Thanks for catching that! 🙏